packages/openapi-generator/src/parser/schema.ts

```diff
@@ -66,6 +66,12 @@ export function parseSchema(
     return parseXOfSchema(schema, refs, 'anyOf', options);
   }
 
+  if (schema.type === 'string' && schema.format === 'binary') {
+    return {
+      type: 'Blob'
+    };
+  }
+
```

Comment:
[pp] I think I would like to push this down into `getType()` (internal) function.

It currently looks like this:
```ts
export function getType(originalType: string | undefined): string {
  const type = originalType ? typeMapping[originalType] : 'any';
  if (!type) {
    logger.verbose(
      `Could not map type '${originalType}' to a native type. Using any.`
    );
    return 'any';
  }
  return type;
}
```

but it might also just take the schema:

```ts
export function getType(schema: Pick<OpenApiSchema, 'type', 'format'>): string {
  if (schema.type === 'string' && schema.format === 'binary') {
    return {
      type: 'Blob'
    };
  }
  const type = originalType ? typeMapping[originalType] : 'any';
  if (!type) {
    logger.verbose(
      `Could not map type '${originalType}' to a native type. Using any.`
    );
    return 'any';
  }
  return type;
}
```

Reply:
I opted to use a union type of schema and string because the type-mapping tests we have would need to be changed otherwise. But the current implementation is a bit cluttered whereas if we just change the tests, it would be cleaner. Thoughts @marikaner?

Reply:
I'd always go for changing the tests rather than keeping cluttered code 😊 This shouldn't affect tooooo many tests, right?

Reply:
Just a little as it turns out. I kept it simple with the type of schema in `getType()`. The reason we need both `NonArraySchemaObject`  and `OpenApiSimpleSchema` is because the function is called in both the parser and file-serializer with different schema typing.